packages/liveblocks-react-ui/src/components/InboxNotification.tsx

```diff
@@ -59,11 +59,8 @@ type ComponentTypeWithRef<
   P,
 > = ComponentType<P & Pick<ComponentProps<T>, "ref">>;
 
-type InboxNotificationKinds = {
-  [K in keyof DAD]: ComponentTypeWithRef<
-    "a",
-    InboxNotificationCustomKindProps<K>
-  >;
+type InboxNotificationKinds<KS extends KDAD = KDAD> = {
```

Comment:
Could you add a comment here to clarify why we need to do that?

Reply:
Added in 5ef77741f60c4c204c512e30901a96df6a9e5ed2 👍